pkgs/misc/tmux-plugins/default.nix

```diff
@@ -70,8 +70,8 @@ in rec {
     src = fetchFromGitHub {
```

Comment:
:arrow_up: Could we update the `version` attribute to reflect the commit date of https://github.com/tmux-plugins/tmux-continuum/commit/26eb5ffce0b559d682b9f98c8d4b6c370ecb639b